x-pack/legacy/plugins/infra/public/pages/logs/log_entry_categories/page_results_content.tsx

```diff
@@ -0,0 +1,248 @@
+      if (selectedTime.isInvalid) {
+        return;
+      }
+      setSelectedTimeRange({
+        startTime: selectedTime.start,
+        endTime: selectedTime.end,
+      });
+      handleQueryTimeRangeChange(selectedTime);
+    },
+    [setSelectedTimeRange, handleQueryTimeRangeChange]
+  );
+
+  const handleAutoRefreshChange = useCallback(
+    ({ isPaused, refreshInterval: interval }: { isPaused: boolean; refreshInterval: number }) => {
+      setAutoRefresh({
+        isPaused,
+        interval,
+      });
+    },
+    [setAutoRefresh]
+  );
+
+  const isFirstUse = useMemo(() => setupStatus === 'hiddenAfterSuccess', [setupStatus]);
+
+  const hasResults = useMemo(() => topLogEntryCategories.length > 0, [
+    topLogEntryCategories.length,
+  ]);
+
+  useEffect(() => {
+    getTopLogEntryCategories();
+  }, [getTopLogEntryCategories, categoryQueryDatasets, categoryQueryTimeRange.lastChangedTime]);
+
+  useEffect(() => {
+    getLogEntryCategoryDatasets();
+  }, [getLogEntryCategoryDatasets, categoryQueryTimeRange.lastChangedTime]);
+
+  useInterval(() => {
+    fetchJobStatus();
+  }, JOB_STATUS_POLLING_INTERVAL);
+
+  useInterval(
+    () => {
+      handleQueryTimeRangeChange({
+        start: selectedTimeRange.startTime,
+        end: selectedTimeRange.endTime,
+      });
+    },
+    autoRefresh.isPaused ? null : autoRefresh.interval
+  );
+
+  return (
+    <ResultsContentPage>
+      <EuiPageBody>
+        <EuiFlexGroup direction="column">
+          <EuiFlexItem grow={false}>
+            <EuiPanel paddingSize="l">
+              <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
+                <EuiFlexItem />
+                <EuiFlexItem grow={false}>
+                  <EuiSuperDatePicker
+                    start={selectedTimeRange.startTime}
+                    end={selectedTimeRange.endTime}
+                    onTimeChange={handleSelectedTimeRangeChange}
+                    isPaused={autoRefresh.isPaused}
+                    refreshInterval={autoRefresh.interval}
+                    onRefreshChange={handleAutoRefreshChange}
+                  />
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </EuiPanel>
+          </EuiFlexItem>
+          {jobHasProblem(jobStatus['log-entry-categories-count'], setupStatus) ? (
+            <EuiFlexItem grow={false}>
+              <LogAnalysisJobProblemIndicator
+                jobStatus={jobStatus['log-entry-categories-count']}
+                onRecreateMlJobForReconfiguration={viewSetupForReconfiguration}
+                onRecreateMlJobForUpdate={viewSetupForUpdate}
+                setupStatus={setupStatus}
+              />
+            </EuiFlexItem>
+          ) : null}
+          {isFirstUse && !hasResults ? (
+            <EuiFlexItem grow={false}>
+              <FirstUseCallout />
+            </EuiFlexItem>
+          ) : null}
+          <EuiFlexItem grow={false}>
+            <EuiPanel paddingSize="l">
```

Comment:
```suggestion
            <EuiPanel paddingSize="m">
```
But I saw that the log rate panels also have large paddings - maybe we should change them together?